### PR TITLE
Front back connect

### PR DESCRIPTION
**Pull request by LauraOD1211**

### CODE REVIEW

File: frontend/app/src/app/feed/feed.component.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ export class FeedComponent implements OnInit {
   getImages(): void {
     this.imageService.getFeed().subscribe((images) => {
       this.images = images;
+      console.log(images)
```
Comment (chripell) [inline frontend/app/src/app/feed/feed.component.ts:23]: Id this a debugging leftover?

**Review by TeoReu — APPROVED** [review]

File: server/routes/images.go [hunk 2]
```diff
@@ -55,6 +58,7 @@ func ImageUploadPage(w http.ResponseWriter, r *http.Request) {
 
 func HandleImageUpload(w http.ResponseWriter, r *http.Request) {
 	ctx := appengine.NewContext(r)
+	fmt.Printf("Called")
```
Comment (chripell) [inline server/routes/images.go:61]: debugging leftover?

File: server/routes/images.go [hunk 3]
```diff
@@ -182,3 +196,44 @@ func UploadImage(ctx context.Context, user_id, image_type string, labels []model
...
+		_, err := it.Next(&image)
+		if err == iterator.Done {
+			break
+		}
+		if err != nil {
+			log.Errorf(ctx, "Error fetching next image: %v", err)
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+		imageURLs = append(imageURLs, "https://storage.cloud.google.com/"+config.ImageBucket()+"/"+image.GCSObjectID())
```
Comment (chripell) [inline server/routes/images.go:228]: Consider using consts for magic strings (like URL in this case)

**Review by chripell — APPROVED** [review]

File: server/routes/images.go (hunk 3 above)
Comment (Omn1) [reply server/routes/images.go:228]: Actually, I feel like making such strings is a responsibility of the StorageClient and you should add smth like `GetServingURL(bucket, object)` to the interface

**Review by Omn1 — APPROVED** [review]

**Pull request merged**